lib/src/clojure_lsp/feature/file_management.clj

```diff
@@ -263,6 +263,10 @@
       (shared/dissoc-in [:analysis filename])
       (shared/dissoc-in [:findings filename])))
 
+(defn ^:private file-deleted [db* uri filename]
+  (swap! db* db-without-file uri filename)
+  (f.diagnostic/publish-empty-diagnostics! uri @db*))
```

Comment:
That's a good pratice, we were really missing that, I noticed some time ago when I would do lots of renames and see old diagnostics

Reply:
oh, we were just missing for file watch, I see

Reply:
Should we implement [`workspace/didDeleteFiles`](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_didDeleteFiles)?